src/psp2_ui.h

```diff
@@ -27,13 +27,9 @@
 #include <vector>
 
 #define SHADERS_NUM 4
-vita2d_shader* shaders[SHADERS_NUM];
-char* shader_names[] = {
-	"None",
-	"Sharp Bilinear",
-	"LCD 3x",
-	"xBR"
-};
+
+extern vita2d_shader* shaders[];
+extern char* shader_names[];
 
```

Comment:
Why are you declaring the shaders as extern? Are they used anywhere outside of psp2_ui.cpp?